src/components/VenueCard.js

```diff
@@ -22,17 +33,35 @@ export const VenueCard = ({ venueName, state, city, url, venuePhone, bookingEmai
           className='emailer'
           >Send Email to {venueName}</a>
         </div>
-      )
-    } else if(info && title !== 'Booking Email'){
+      );
+    } else if (classTitle === 'website'){
+      return  info !== '' && isUrlValid(info) === true
+        ?
+        <a
+          target="_blank"
+          href={url}
+          className={classTitle}
+          rel="noopener noreferrer"
+        >{url}</a>
+        :
+        <a
+          target='_blank'
```

Comment:
Nice - I couldn't tell from the screenshot if the search results opened in a separate tab. Is this just extra helpfulness for the user so they can gather extra info/locations/etc? Or does something they do in the search results tie back into the app?